static/js/services/languages.js

```diff
@@ -0,0 +1,20 @@
+var _ = require('underscore');
+
+/*
+ * Get all enabled languages
```

Comment:
Can you put that in `language.js` instead? There's a bunch of language-related services there already.

(And I also find it a bit weird to have these small one-off services, rather than `Language.setLanguage()` and `Language.enabledLanguages()` for instance, but it's personal taste I suppose.)